server/src/main/java/io/druid/segment/realtime/plumber/Plumber.java

```diff
@@ -44,6 +47,7 @@
    *
    * @param commitRunnable code to run after persisting data
    */
+  void persist(Committer commitRunnable);
   void persist(Runnable commitRunnable);
```

Comment:
We did it this way for backwards compatibility and I think it's required for that.